sources/10-091r3/sections/10-references.adoc

```diff
@@ -10,53 +10,73 @@
 
 * [[[ogc06-121r3, OGC 06-121r3]]], OGC 06-121r3, _OGC Web Services Common Specification_
 
-* [[[nasa1, NASA ESDS-RFC-011v2.00 R. Rew]]], E. Hartnett, D. Heimbigner, E. Davis, J. Caron: _NetCDF Classic and 64-bit Offset File Formats_ +
-http://www.esdswg.org/spg/rfc/esds-rfc-011/ESDS-RFC-011v2.00.pdf
-
-* [[[netcdf_refdoc, Unidata UCAR]]], NetCDF Reference Document, 2009 +
+* [[[netcdf_refdoc,1]]] Unidata UCAR, NetCDF Reference Document, 2009 +
```

Comment:
Let's remove all the line breaks -- they should not exist in a bibliography.

Reply:
Changes applied.